src/System.Collections/src/System/Collections/BitArray.cs

```diff
@@ -341,6 +341,123 @@ public BitArray Not()
             return this;
         }
 
+        /*=========================================================================
+        ** Shift all the bit values to right on count bits. When count less zero it
+        ** shift all the bit values to left. New bits are be set to zero. The 
+        ** current instance is updated and returned.
+        =========================================================================*/
+        public BitArray RightShift(int count)
+        {
+            if (count <= 0)
+            {
+                if (count == 0)
+                {
+                    _version++;
+                    return this;
+                }
+                count = -count;
+                // Following trick reduce check with int.MinValue
+                if (unchecked((uint)count < (uint)m_length))
+                    return LeftShift(count);
+            }
+
+            int toIndex = 0;
+            int ints = GetArrayLength(m_length, BitsPerInt32);
+            if (unchecked((uint)count < (uint)m_length))
+            {
+                var fromIndex = count / BitsPerInt32;
+                int shiftCount = count % BitsPerInt32;
```

Comment:
@AlexRadch I guess that method is not exposed in whatever targetframework the repo is targeting. Either that or the dependency chain is such that BitArray is lower in the dependency chain than the Math implementation. I seriously doubt that though.

I see you've inlined the implementation. I'm not sure that's a good idea as the jitter should eventually recognized div followed by rem and use the results from the hardware immediately (which is why the change to Math.DivRem is temporary). 

In any case, I don't think its a big deal either way. Sorry for the confusion.